standard/links/commons/links_match_priority_groups.lua

```diff
@@ -0,0 +1,22 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Links/MatchPriorityGroups
```

Comment:
Given there is currently just one order and no groups, perhaps
```suggestion
-- page=Module:Links/MatchPriority
```

Reply:
no opinion on the name